website/guides/terraform-provider-development-program.html.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "guides"
+page_title: "Terraform Provider Development Program"
+sidebar_current: "guides-terraform-provider-development-program"
+description: This guide is intended for vendors who're interested in having their platform supported by Teraform. The guide walks vendors through the steps involved in creating a provider and applying for it to be included with Terraform.
+---
+
+# Terraform Provider Development Program
+
+The Terraform Provider Development Program allows vendors to build
+Terraform providers that are officially approved and tested by HashiCorp and
+listed on the official Terraform website. The program is intended to be largely
+self-serve, with links to information sources, clearly defined steps, and
+checkpoints.
+
+-> **Building your own provider?** If you're building your own
+provider and aren't interested in having HashiCorp officially maintain
+the provider, refer to the
```

Comment:
Can we change this

    in having HashiCorp officially maintain the provider 

to 

    in having HashiCorp officially approve and regularly test the provider

or similar? We're implying that this path leads to us maintaining it.

It was my understanding that we're still looking for the "Vendor" behind this new Provider to do the maintenance here, right? Below, we detail a `Support` section about our expectations of support, but this line here suggests HashiCorp will (or will be involved) in the direct support, which I don't think was the intent

Reply:
May also want to mention the "Self host" part of building your own provider. The idea being that `github.com/terraform-providers` is only official, approved, and tested providers.

Reply:
Done. I think the self host part is a good point in should go in the "writing your own provider" part. I'll update that